_security/access-control/anonymous-authentication.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: Anonymous authentication
+parent: Access control
+nav_order: 145
+---
+
+# Anonymous authentication
+
+The Security plugin supports anonymous authentication where a user is able to access a cluster without providing credentials. This is useful in cases where you want lots of people to be able to access your cluster with a common set of privileges. 
+
+## Configuration
+
+To enable anonymous authentication, you need to modify the `config.yml` file inside the `opensearch-security` configuration subdirectory of your cluster.
+
+In the `config.yml` file, there is an `http` section which includes the `anonymous_auth_enabled` setting: 
+
+```yml
+http:
+  anonymous_auth_enabled: <true|false>
+  ...
+```
+
+The following table describes the `anonymous_auth_enabled` setting. For more information, see the [configuration]({{site.url}}{{site.baseurl}}/security/configuration/configuration/) file overview. 
+
+| Setting | Description |
+| :--- | :--- |
+| `anonymous_auth_enabled` | Either enables or disables anonymous authentication. When you enable anonymous authentication, all defined HTTP authenticators are non-challenging. See [The challenge setting]({{site.url}}{{site.baseurl}}/security/authentication-backends/basic-authc/#the-challenge-setting). |
```

Comment:
Where should the HTTP authenticators link to? Is there a doc page that describes them?

Reply:
The Http Authenticators are described in the general authentication backend page.

Reply:
Thanks; we're already linking to that page so it should be good then.